tests/posix/common/testcase.yaml

```diff
@@ -2,7 +2,7 @@ common:
   arch_exclude: posix
   tags: posix
   min_ram: 64
-  timeout: 120
+  timeout: 600
```

Comment:
Ouch.  Is 10 minutes really needed?

Reply:
On my modest setup, sometimes it fails with timeout if less time is used. An alternative would be to reduce the [BOUNCES count](https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/posix/common/src/pthread.c#L19) for this platform to an acceptable value so the test still makes sense. Although an increase in the timeout would be still needed I think.

Reply:
I encountered a similar issue, you may ref https://github.com/zephyrproject-rtos/zephyr/pull/40575 to add a platform-specific test timeout, thus your change will not affect other platforms.